package.json

```diff
@@ -32,6 +32,7 @@
     ]
   },
   "devDependencies": {
+    "axios": "^0.18.0",
```

Comment:
Should not `axios` be in the `express` package.json instead?

Reply:
My approach was to move any `devDependency` that is (or will) be used in more than one repo to the top. Another small thing I'd like to do eventually was to unify the HTTP lib (currently there  `axios` and `request` seem to be used)

Reply:
I think that package devDependencies should remain in packages, and be added to the top if they are used outside these packages.

To avoid having duplicates, you can use lerna's '--hoist' option, and you'll go faster by using yarn instead of npm.

Reply:
Makes sense, I started this because they recommended it in the Lerna Readme at https://github.com/lerna/lerna#common-devdependencies

Reply:
The reasons given are no longer valid since there is the `--hoist` option and the yarn workspaces.